microsoft-edge/extensions-chromium/publish/aad-account.md

```diff
@@ -0,0 +1,67 @@
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer
+---
+
+# Add and manage users from your organization on to the Microsoft Edge program
+
+To add and manage users in the Microsoft Edge program to manage extensions, you can now associate your Partner Center account with your organization's Azure Active Directory.
+
+## Step 1: Enroll in the Microsoft Edge program on Partner Center 
+
+1. If you are new to Partner Center and don't have an account or if you have an existing commercial account with Partner Center (see step **2.c** below), create a new [Microsoft account (MSA)][WindowsCommunityEverythingAboutMicrosoftAccounts] (or use an existing one) to register with the Microsoft Edge program. Follow the steps in the [Enroll in the Microsoft Edge program for Partner Center][DeveloperRegistration]. 
+
+1. If you have a developer account registered on Partner Center, use the corresponding Microsoft account (MSA) to sign in to your developer account. Then enroll in the Microsoft Edge program. The Microsoft Edge program on the Partner Center **does not support enrolling directly** with a work or school account. 
+    1. **If you are already signed in to Partner Center using your work account**, you need to sign in as a Global Admin or Primary Owner. If you are not the owner, navigate to [User management][https://partner.microsoft.com/dashboard/account/v3/usermanagement] to find the primary owner of the account.
+    1. Verify that the primary owner is an MSA (Hotmail/Live/Outlook) user. Ask the account owner to register for the Microsoft Edge Add-ons program using the steps at [Enroll in the Microsoft Edge program for Partner Center][DeveloperRegistration].
+    1. If your organization does not have an MSA (Hotmail/Live/Outlook) account as a primary owner, you need to register for the Microsoft Edge program with an MSA account. This happens when your organization has a commercial account on Partner Center. Follow the steps at [Enroll in the Microsoft Edge program for Partner Center][DeveloperRegistration].
+
+Once the account is set up, continue to the next section to link your Azure AD tenants with this account for extension management.
+
+## Step 2: Associate Azure Active Directory with your account
+
+You can use Azure Active Directory to add and manage users in the Microsoft Edge Program account. You can add individual users, groups of users, or Azure AD applications. 
+
+### Associate Azure Active Directory with your account
+
+In order to add and manage account users, you must first associate your account with your organization's Azure Active Directory. If your organization already uses Office 365 or other business services from Microsoft, you already have Azure AD. Otherwise, you can [create a new Azure AD tenant][../../../windows/uwp/publish/associate-azure-ad-with-partner-center#create-a-brand-new-azure-ad-to-associate-with-your-partner-center-account] here for free.
```

Comment:
Can we fix the link?

Reply:
Can we replace the above text with:
Otherwise you can create a new Azure AD tenant for free. For more information, navigate to [pagename][link]

Reply:
fixed both